src/components/Header/Header.jsx

```diff
@@ -70,57 +74,58 @@ const DropdownMenuItem = styled(MenuItem)`
 `;
 
 const ToolbarHeader = styled(Toolbar)`
-  background-color: ${constants.defaultPrimaryColor} !important;
-  padding: 8px !important;
+  backdrop-filter: blur(16px);
+  background-color: ${constants.defaultPrimaryColorSolid} !important;
+  height: 56px;
+  left: 0;
+  padding: 8px 16px !important;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 100;
+
   & a {
     color: ${constants.primaryTextColor};
+
     &:hover {
       color: ${constants.primaryTextColor};
       opacity: 0.6;
     }
   }
 `;
 
-const LinkGroup = ({ navbarPages }) => {
-  const router = useReactRouter();
-  const [anchorEl, setAnchorEl] = useState(null);
-  const handleClose = useCallback(() => {
-    setAnchorEl(undefined);
-  }, [anchorEl]);
+const MenuContent = styled.div`
+  background: ${constants.primarySurfaceColor};
+  height: 100%;
+  max-width: 300px;
+  min-width: 220px;
+`;
+
+const MenuLogoWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  padding: 24px 0;
+`;
+
+const DrawerLink = styled(Link)`
+  color: ${constants.textColorPrimary};
+`;
+
+const LinkGroup = () => {
+  const navigationState = useContext(navigationContext);
 
   return (
     <VerticalAlignToolbar>
-      {navbarPages.slice(0, 6).map(page => (
+      {navigationState.navbarPages.map(page => (
         <TabContainer key={page.key}>
           <Link to={page.to}>{page.label}</Link>
         </TabContainer>
       ))}
-      <TabContainer>
-        <IconButton size="small" color="inherit" onClick={e => setAnchorEl(e.currentTarget)}>
-          <MoreVert />
-        </IconButton>
-        <DropdownMenu anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={handleClose}>
-          {navbarPages.slice(6, navbarPages.length).map(page => (
-            <DropdownMenuItem
-              onClick={() => {
-                router.history.push(page.to);
-                handleClose();
-              }}
-              key={page.key}
-            >
-              {page.label}
-            </DropdownMenuItem>
-          ))}
-        </DropdownMenu>
-      </TabContainer>
     </VerticalAlignToolbar>
   );
 };
 
-LinkGroup.propTypes = {
-  navbarPages: PropTypes.arrayOf(PropTypes.shape({})),
-};
-
 const SettingsGroup = ({ children }) => {
   const [anchorEl, setAnchorEl] = useState(null);
```

Comment:
Here I set an anchorEl state which I will use to define were the dropdown menu should be attached.